test_olog.py

```diff
@@ -141,3 +181,25 @@ def test_put_properties():
 @vcr.use_cassette()
 def test_put_property():
     cli.put_property(PROPERTY)
+
+
+@vcr.use_cassette()
+def test_put_property_with_error():
+    # vcr will return a wrong logbook
```

Comment:
```suggestion
    # vcr will return a wrong logbook because the recorded
    # response has been manually edited to be inconsistent
    # with the request to exercise this code path
```